src/plugins/embeddable/public/lib/attribute_service/attribute_service.tsx

```diff
@@ -181,7 +181,7 @@ export class AttributeService<
         this.showSaveModal(
           <SavedObjectSaveModal
             onSave={onSave}
-            onClose={() => reject()}
+            onClose={() => {}}
```

Comment:
Since `showSaveModal` used to discard the `onClose` prop but now calls it, I've replaced this with an empty callback to ensure `reject` doesn't get called. This seems to be the only reference to `showSaveModal` that didn't already pass an empty `onClose` callback to its modal anyway.